### PR TITLE
Modify building daal4py for wheel packages

### PR DESCRIPTION
**Pull request by Pahandrovich**
## Description
Daal4py from pip, without conda environment can't search onedal dll libraries. This changes needed to fix this problem.

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Cython==0.29.21
-Jinja2==2.11.3
-numpy==1.20.0
-clang-format==9.0.0
+Cython==0.29.21
+Jinja2==2.11.3
+numpy>=1.18
```
Comment (SmirnovEgorRu) [inline requirements-dev.txt:3]: Downgrade?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+dpcpp_cpp_rt==2021.1
+daal==2021.1
+numpy==1.20
```
Comment (PetrovKP) [inline requirements.txt:3]: Users can use numpy less:
```suggestion
numpy>=1.15
```

```

File: setup.py [hunk 3]
```diff
@@ -386,12 +371,32 @@ def gen_pyx(odir):
...
+
 # daal setup
 setup(name="daal4py",
       description="A convenient Python API to Intel(R) oneAPI Data Analytics Library",
+      long_description=long_description,
+      license="Apache-2.0",
       author="Intel",
       version=d4p_version,
       url='https://github.com/IntelPython/daal4py',
+      author_email="onedal.maintainers@intel.com",
```
Comment (PetrovKP) [inline setup.py:398]: Need to align with current attributes

File: setup.py [hunk 4]
```diff
@@ -386,12 +371,32 @@ def gen_pyx(odir):
...
+            if "dpcpp_cpp_rt" in r:
+                install_requires.remove(r)
+                break
+
+setup_requires = []
+with open('requirements-dev.txt') as f:
+    setup_requires.extend(f.read().splitlines())
+
 # daal setup
 setup(name="daal4py",
```
Comment (PetrovKP) [inline setup.py:391]: I would like see a result to tets.pypi

**Review by PetrovKP — APPROVED** [review]

**Pull request merged**